index.js

```diff
@@ -531,6 +531,14 @@ function Argv (processArgs, cwd) {
     return argv
   }
 
+  function guessLocale () {
+    try {
+      return osLocale.sync()
```

Comment:
two options I see:

1. is there an environment variable I can detect and flip this feature off specifically for Atom.
2. what if I add an option `.detectLocale(false)`?

Shall I release this in the interim to fix the exception?